File: backend/src/SmartHome.Domain/Simulations/SimulationSpeed.cs
using System.ComponentModel;

namespace SmartHome.Domain.Simulations;

/// <summary>
/// Speed multipliers controlling how fast the ambient temperature simulation advances.
/// </summary>
public enum SimulationSpeed
{
    /// <summary>Normal speed — one simulation tick every 5 seconds.</summary>
    [Description("1x")] OneX = 1,
    /// <summary>2× speed — one simulation tick every 2.5 seconds.</summary>
    [Description("2x")] TwoX = 2,
    /// <summary>5× speed — one simulation tick every 1 second.</summary>
    [Description("5x")] FiveX = 5,
    /// <summary>10× speed — one simulation tick every 0.5 seconds.</summary>
    [Description("10x")] TenX = 10
}

File: frontend/smart-home-ui/src/app/types/simulationspeed.ts
/** Simulation speed multipliers controlling how fast simulation time advances. */
export enum SimulationSpeed {
  /** Normal speed — 1 second of simulation per real second. */
  OneX = 'OneX',
  /** 2× speed — simulation advances twice as fast as real time. */
  TwoX = 'TwoX',
  /** 5× speed — simulation advances five times faster than real time. */
  FiveX = 'FiveX',
  /** 10× speed — simulation advances ten times faster than real time. */
  TenX = 'TenX',
}
